internal/sirius/edit_dates_test.go

```diff
@@ -40,9 +40,6 @@ func TestEditDates(t *testing.T) {
 					}).
 					WillRespondWith(dsl.Response{
 						Status: http.StatusOK,
-						Body: dsl.Like(map[string]interface{}{
-							"rejectedDate": dsl.String("04/03/2022"),
-						}),
```

Comment:
Took this out because the body _doesn't_ include `rejectedDate` and the Pact tests are failing (not noticed previously because a webhook was broken). Since we don't actually use the rejectedDate, there's no need to reason it in the contract.